src/logic/operate.test.js

```diff
@@ -0,0 +1,5 @@
+import operate from './operate';
+
+it('Operate should work without error', () => {
+  expect(operate.length).toBe(3);
+});
```

Comment:
- [ ] All the functionalities & scenarios should be tested, e.g, test when operation is `+`, `-`, `x`, `÷`, `%`,  basically you should test all possible scenarios. Please add more tests.

Reply:
Thank you. The test has been expanded.